tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1307,6 +1307,112 @@ def _reset_allow_build_at_runtime(node):
     signatures[self._input_saved_model_signature_key] = self._converted_func
     save.save(self._saved_model, output_saved_model_dir, signatures)
 
+  def summary(self, line_length=160, detailed=True, print_fn=None):
+    """This method describes the results of the conversion by TF-TRT. It
+    includes information such as the name of the engine, the number of nodes per
+    engine, the input and output dtype, along with the input shape of each
+    TRTEngineOp.
+
+    Args:
+      line_length: Default line length when printing on the console.
```

Comment:
I can think about 2 alternatives:
- stop asking the user for a `line_length` and use `os.get_terminal_size()`. Hopefully it will give us some sensible numbers.
- we stop using % of the line and we fix the numbers of characters that can be printed per column 

@bixia1 : any other idea ? What would you prefer ? Keras implemented this exactly the way I did, hence I did it that way. No verification over `line_length`

Reply:
Fixed with setting a minimum acceptable value